commons/gridgo-bean/src/main/java/io/gridgo/bean/pojo/BElementTranslators.java

```diff
@@ -117,7 +121,11 @@ public static BContainer toBContainer(GenericData ele, PojoMethodSignature signa
         if (ele.isReference())
             return BObject.ofPojo(ele.asReference().getReference());
 
-        throw new IllegalArgumentException("Expect key-value or sequence or reference data, got " + ele);
+        if (signature == null)
```

Comment:
checks need to be consistent across methods. should refactor in a common methods